libs/atqr/domain/src/lib/domain-spec/services/Deadline Monitor.disl

```diff
@@ -13,106 +13,92 @@ domain ATQR
        */
       ⏰ NewDayStarted!
         for each :challenge in Challenges
-          when .deadline is before Today
-            publish DeadlineReached! :challengeId
+          when .deadline is after Today
```

Comment:
in the “Challenge Aggregate.disl" file there is an event called: “DeadlineReached” that does the same check. Select which one should remain in the documentation.